client/src/Component/Main/Main.jsx

```diff
@@ -0,0 +1,72 @@
+import "./main.scss";
+// import Data from "../../data/exampleresponse.json";
+import VideoCard from "./VideoCard";
+import AddVideo from "./AddVideo";
+import { useState, useEffect } from "react";
+
+const Videos = () => {
+  const [videosData, setVideoData] = useState([]);
+
+  // const addVideo = (formValue) => {
+  //   setVideoData([...videosData, formValue]);
+  // };
```

Comment:
Best to try and delete code that you dont need anymore, it can get a bit of a mess!

Reply:
I removed them